core-ui/src/components/Predefined/Details/Namespace/CreateWorkloadForm/BasicData.js

```diff
@@ -47,12 +47,11 @@ export default function BasicData({ deployment, setDeployment }) {
       <h3 className="configuration-data__title">Service options</h3>
       <Checkbox
         defaultChecked={deployment.createService}
-        value="Create Service"
         onChange={e =>
           setDeployment({ ...deployment, createService: e.target.checked })
         }
       >
-        Create Service
+        Create a separate Service resource to expose this Deployment
```

Comment:
I think this would be enough
```suggestion
        Create a Service to expose the Deployment
```

Reply:
I had a conversation with @klaudiagrz about this change. I wanted to emphasize that a separate resource will be created despite the fact user is on the **deployment** creation form.

Reply:
...and @klaudiagrz approved this.